api/views/workflow.py

```diff
@@ -36,7 +36,7 @@ class ProviderViewSet(viewsets.ReadOnlyModelViewSet):
     serializer_class = ProviderSerializer
 
     def get_queryset(self):
-        queryset = ShareUser.objects.exclude(robot='').exclude(long_title='')
+        queryset = ShareUser.objects.exclude(long_title='').exclude(favicon='')
```

Comment:
Robots have `is_trusted=False` but theoretically push users who are not trusted should not have favicons. Not filtering out `is_trusted=False` also leaves the flexibility of accepting changes from a user who is not trusted and giving them a favicon so that they will show up as a source. Any flaws in that logic?